server/auth/name-space/name-space.guard.ts

```diff
@@ -3,13 +3,14 @@ import {
     ExecutionContext,
     Injectable,
     NotFoundException,
+    Scope,
     UnauthorizedException,
 } from "@nestjs/common";
 import { ConfigService } from "@nestjs/config";
 import { Configuration, FrontendApi } from "@ory/client";
 import { NameSpaceService } from "./name-space.service";
 
-@Injectable()
+@Injectable({ scope: Scope.REQUEST })
```

Comment:
suggestion: I think this is a good change, but make sure this is will tested across all endpoints.